doc/sphinx/proof-engine/ltac.rst

```diff
@@ -2011,7 +2011,7 @@ Proving that a list is a permutation of a second list
 
    .. coqtop:: none
 
-      Require Import List.
+      Require Import ListDef.
```

Comment:
That's not a renaming. That's because the `List` module is moved to the stdlib directory, but this particular example only depended on `ListDef`, which happens to be in `rocq-init`.